doc/rados/configuration/mclock-config-ref.rst

```diff
@@ -625,7 +625,7 @@ Steps to Manually Benchmark an OSD (Optional)
 
 .. tip:: If you have already determined the benchmark data and wish to manually
          override the max osd capacity for an OSD, you may skip to section
-         `Specifying  Max OSD Capacity`_.
+         `OSD Capacity Determination (Automated)`_.
```

Comment:
The correct section title is "Set or Override Max IOPS Capacity of an OSD". Please change it accordingly.